components/sup/plan.sh

```diff
@@ -33,6 +34,7 @@ _common_prepare() {
   build_line "Setting CARGO_TARGET_DIR=$CARGO_TARGET_DIR"
 }
 
+# shellcheck disable=2155
```

Comment:
I'm assuming plan.sh is never going to be processed with `set -e`, or else we should probably fix these up per https://github.com/koalaman/shellcheck/wiki/SC2155